CHANGELOG_PENDING.md

```diff
@@ -47,6 +47,7 @@ Special thanks to external contributors on this release:
 ### FEATURES
 
 - [rpc] [\#7270](https://github.com/tendermint/tendermint/pull/7270) Add `header` and `header_by_hash` RPC Client queries. (@fedekunze)
+- [rpc] Add `ApplicationInfo` to `status` rpc call which contains the application version. (@jonasbostoen)
```

Comment:
```suggestion
- [rpc] [\#7701] Add `ApplicationInfo` to `status` rpc call which contains the application version. (@jonasbostoen)
```

Reply:
You'll need to leave off the brackets around the PR, as I discovered recently. 🙂